spec/dummy/config/environments/test.rb

```diff
@@ -40,4 +47,9 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  # Time columns will become time zone aware in Rails 5.1.
+  if Rails.version.split('.').first.to_i >= 5
```

Comment:
I think there is a way better than this. For example it's to use `Rails::VERSION::MAJOR`.